units/systemd-machined.socket

```diff
@@ -0,0 +1,25 @@
+#  SPDX-License-Identifier: LGPL-2.1-or-later
+#
+#  This file is part of systemd.
+#
+#  systemd is free software; you can redistribute it and/or modify it
+#  under the terms of the GNU Lesser General Public License as published by
+#  the Free Software Foundation; either version 2.1 of the License, or
+#  (at your option) any later version.
+
+[Unit]
+Description=Virtual Machine and Container Registration Service Socket
+Documentation=man:systemd-machined.service(8)
+DefaultDependencies=no
+Conflicts=shutdown.target
+Before=sockets.target shutdown.target
+RequiresMountsFor=/var/lib/machines
```

Comment:
is this necessary for socket too? If this is e.g. automount we can activate it only when service is started?

Reply:
So in systemd-import.socket has `WantsMountsFor=/var/lib/machines /var/lib/portables /var/lib/extensions /var/lib/confexts` which is what inspired me to add this.

Reply:
Um, that one has DefaultDependencies=no, and carries explicit ordering against remount-fs.

This here probably doesn't hurt, I just don't see the need I guess